risc0/zkvm/src/host/receipt.rs

```diff
@@ -159,11 +147,11 @@ pub struct Receipt {
     pub journal: Vec<u8>,
 }
 
-/// An inner receipt can take the form of a collection of [SegmentReceipts] or a
+/// An inner receipt can take the form of a collection of [SegmentReceipt]s or a
 /// [SuccinctReceipt].
 #[derive(Debug, Deserialize, Serialize, PartialEq)]
 pub enum InnerReceipt {
-    /// The [SegmentReceipts].
+    /// The [SegmentReceipt]s.
```

Comment:
This is wrong, it actually is `[SegmentReceipts]` which is distinct from `[SegmentReceipt]`